input/resources/StructureDefinition-profile-SignalUSCoreOrganizationProfile.json

```
@@ -53,6 +53,24 @@
         ],
         "isModifier": false
       },
+      {
+        "id": "Organization.identifier:EIN",
+        "path": "Organization.identifier",
+        "sliceName": "EIN"
+      },
```

Want additional information about each slice:
```suggestion
        "short": "Employer Identification Number (EIN)",
        "definition": "An Employer Identification Number (EIN) is also known as a Federal Tax Identification Number, and is used to identify a business entity. Generally, businesses need an EIN. Provided by the US Internal Revenue Service (IRS)",
        "comment": "An Employer Identification Number (EIN), also known as a Federal Tax Identification Number, is used to identify a business entity.",
      },
```